ui/js/services/jobfilters.js

```diff
@@ -476,6 +477,11 @@ treeherder.factory('thJobFilters', [
                 _.isEqual(_toArray($location.search()[QS_CLASSIFIED_STATE]), ['unclassified']));
     };
 
+    var _isInProgressShown = function() {
+        return (_.contains(_getFiltersOrDefaults('filter-resultStatus'), 'pending') || 
+                _.contains(_getFiltersOrDefaults('filter-resultStatus'), 'running'))
+    }
+
     var _matchesDefaults = function(field, values) {
         $log.debug("_matchesDefaults", field, values);
```

Comment:
Ends up this is pretty brittle.  This function always wants the ``field`` value to have no previx.  So we should add this line here:

    field = _withoutPrefix(field);